### DISCUSSION TITLE
Horizontal width of the console font?

### DISCUSSION THREAD

**Discussion by Vctr43** [microsoft/vscode-cpptools#11629] [Q&A]
I want to make ASCII lines in the console, but to do it perfect i need the ratio of the lenght and the width of every character.
Which is the ratio of the console font? or which is the lenght and width?

**Comment by bobbrow** [comment]
I'm sorry, I don't think that anyone in this repo knows the answer to your question. Our extension does not participate in the font rendering aspect of VS Code. You might get better results with a search engine.
